frontend/src/constants/message.ts

```diff
@@ -42,6 +42,7 @@ const ERROR_MESSAGES = {
   POST_REVIEW_COMPLETE: "코드리뷰 완료 요청에 실패했습니다.",
   GET_MY_REVIEWERS: "리뷰어 목록을 불러오는 도중 에러가 발생하였습니다.",
   GET_MY_REVIEWEES: "리뷰이 목록을 불러오는 도중 에러가 발생하였습니다.",
+  POST_REVIEW_URGE: "코드리뷰 재촉하기에 실패했습니다.",
```

Comment:
```suggestion
  POST_REVIEW_URGE: "코드리뷰 재촉 요청에 실패했습니다.",
```
코드리뷰 완료랑 비슷하게 이런 문구 어떤가요~~?